src/app/result/[postId]/_components/ProfileBlock.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+
+interface Props {
+  name: string;
+  imageUrl: string;
+}
+
+const ProfileBlock = ({ name, imageUrl }: Props) => {
+  return (
+    <div className='flex w-full items-center justify-start gap-4 p-4'>
```

Comment:
```suggestion
    <div className='flex w-full items-center gap-4 p-4'>
```
다음처럼 수정해도 좋을 것 같아요 !